samples/RefreshFlow/AngularApp/src/test.ts

```diff
@@ -0,0 +1,32 @@
+import './polyfills.ts';
```

Comment:
There aren't any tests specifically for the sample, those are simply the files create by Angular CLI when the project created.
I'll remove them anyways